components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1552,36 +1520,88 @@
+<dt_remap_factor  dyn_target="preqx">  -1  </dt_remap_factor>
+<hypervis_subcycle_q dyn_target="preqx"> 1 </hypervis_subcycle_q>
+<hypervis_subcycle_tom dyn_target="preqx"> 0 </hypervis_subcycle_tom>
+
+<qsplit dyn_target="preqx">  1 </qsplit>
+
+<rsplit dyn_target="preqx" hgrid="ne4np4" >   2 </rsplit>
+<rsplit dyn_target="preqx" hgrid="ne11np4" >  2 </rsplit>
+<rsplit dyn_target="preqx">  3 </rsplit>
+
+<se_nsplit dyn_target="preqx" hgrid="ne4np4" >   2 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne11np4" >  4 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne16np4" >  1 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne30np4" >  2 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne60np4" >  4 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne120np4">  4 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne240np4">  5 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne512np4">  5 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne1024np4"> 10 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_arm_x8v3_lowcon"  > 5 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_conus_x4v1_lowcon"  > 4 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_northamericax4v1"  > 4 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_svalbard_x8v1_lowcon"  > 5 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_sooberingoa_x4x8v1_lowcon"  > 5 </se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_enax4v1">4</se_nsplit>
+<se_nsplit dyn_target="preqx" hgrid="ne0np4_twpx4v1" > 4 </se_nsplit>
+<se_nsplit dyn_target="preqx"> 2 </se_nsplit>
+
+<hypervis_subcycle hgrid="ne4np4" dyn_target="preqx">   3 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne11np4" dyn_target="preqx" >  3 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne16np4" dyn_target="preqx">  3 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne30np4" dyn_target="preqx">  3 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne60np4" dyn_target="preqx">  4 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne120np4" dyn_target="preqx">  4 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne240np4" dyn_target="preqx">  4 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne512np4" dyn_target="preqx">  10 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne1024np4" dyn_target="preqx"> 20 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_arm_x8v3_lowcon" dyn_target="preqx" > 8 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_conus_x4v1_lowcon" dyn_target="preqx" > 7 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_northamericax4v1" dyn_target="preqx"  > 7 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_svalbard_x8v1_lowcon" dyn_target="preqx" > 8 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_sooberingoa_x4x8v1_lowcon" dyn_target="preqx"  > 8 </hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_enax4v1" dyn_target="preqx">7</hypervis_subcycle>
+<hypervis_subcycle hgrid="ne0np4_twpx4v1" dyn_target="preqx"> 7 </hypervis_subcycle>
```

Comment:
There are a lot of changes in resolutions that are not tested nightly, like ne120 and RRM with preqx dycore. How would we know settings for them are not modified?

Reply:
for all grid specific settings, I added a dyn_target="preqx" clause, so they will all be preserved for the preqx dycore.  The the grid independent settings, the integration suite should be a good test of if any preqx settings are changed.

Reply:
Yes, some changes can be read like that, that preqx dycore was added, but some other changes are harder to parse.